regression/benchmark_scripts/benchmark.py

```
@@ -0,0 +1,346 @@
+    base_mr = rmm.mr.get_current_device_resource()
+    stats_mr = rmm.mr.StatisticsResourceAdaptor(base_mr)
+    rmm.mr.set_current_device_resource(stats_mr)
+
+    return base_mr, stats_mr
+
+
+def worker(gpu_id, dataset_file_path, csv_path, git_commit, log_path, test_status_file, n_gpus, d_type="routing"):
+    import os
+    os.environ["CUDA_VISIBLE_DEVICES"] = gpu_id
+
+    import glob
```

Flake8 (7.3.0)

[error] redefinition of unused 'glob' from line 20

(F811)